libs/ui-lib/lib/common/components/clusterConfiguration/ControlPlaneNodesDropdown.tsx

```diff
@@ -0,0 +1,90 @@
+import * as React from 'react';
+import { Dropdown, DropdownItem, DropdownToggle, FormGroup, Tooltip } from '@patternfly/react-core';
+import { useTranslation } from '../../hooks/use-translation-wrapper';
+import { getFieldId, StaticField } from '../..';
+import { useField } from 'formik';
+
+interface ControlPlaneNodesOption {
+  value: string;
+  label: string;
+}
+
+const isDropdownItemEnabled = (controlPlaneNodeCount: string, isNutanix?: boolean): boolean => {
+  return (controlPlaneNodeCount === '1' && !isNutanix) || controlPlaneNodeCount !== '1';
+};
+
+const ControlPlaneNodesDropdown = ({
+  isDisabled = false,
+  isNutanix,
+}: {
+  isDisabled?: boolean;
+  isNutanix?: boolean;
+}) => {
+  const { t } = useTranslation();
+  const [{ name, value }, , { setValue }] = useField<string | ''>('controlPlaneAgents');
```

Comment:
Im not sure what benefit we get using `string || ''` instead of just `string` ?
```suggestion
  const [{ name, value }, , { setValue }] = useField<string>('controlPlaneAgents');
```